poll.go

```diff
@@ -108,12 +108,12 @@ func pollRelease(runner Runner, releaseName string, namespace string, timeout in
 	return Release{}
 }
 
-func parseArgs() (*string, *string, *int, *int, *bool) {
+func parseArgs() (*string, *string, *int, *int, *string) {
 	optRelease := getopt.StringLong("release", 'r', "", "Release name to poll for.")
-	optNamespace := getopt.StringLong("namespace", 'n', "default", "Namespace where the release is installed. (default: \"default\")")
+	optNamespace := getopt.StringLong("namespace", 'n', os.Getenv("HELM_NAMESPACE"), "Namespace where the release is installed. (default: \"default\")")
 	optTimeout := getopt.IntLong("timeout", 't', 300, "The timeout in seconds (default: 300)")
 	optInterval := getopt.IntLong("interval", 'i', 5, "The polling interval in seconds (default: 5)")
-	optDebug := getopt.BoolLong("debug", 0, "Run with debug messages on")
+	optDebug := getopt.StringLong("debug", 0, os.Getenv("HELM_DEBUG"), "Run with debug messages on")
```

Comment:
BoolLong does not support a default value :O which is why I had to rely on StringLong to parse the env var :(